docs/docs/changelog.md

```diff
@@ -3,6 +3,60 @@ title: Changelog
+
+#### 🐛 Bug Fixes
+
+- [[#3862](https://github.com/airyhq/airy/issues/3862)] Fixed icons and improved animations [[#3863](https://github.com/airyhq/airy/pull/3863)]
+- [[#3835](https://github.com/airyhq/airy/issues/3835)] Fix unread counter crash [[#3836](https://github.com/airyhq/airy/pull/3836)]
+- [[#3820](https://github.com/airyhq/airy/issues/3820)] Fixes for Cognigy AI on Control Center [[#3821](https://github.com/airyhq/airy/pull/3821)]
+- [[#3813](https://github.com/airyhq/airy/issues/3813)] Fixes to SearchBar on the Control Center UI  [[#3818](https://github.com/airyhq/airy/pull/3818)]
+- [[#3804](https://github.com/airyhq/airy/issues/3804)] Fix using existing subnets for AWS [[#3812](https://github.com/airyhq/airy/pull/3812)]
+- [[#3800](https://github.com/airyhq/airy/issues/3800)] Minor fixes [[#3803](https://github.com/airyhq/airy/pull/3803)]
+
+#### 📚 Documentation
+
+- [[#3801](https://github.com/airyhq/airy/issues/3801)] Cognigy AI docs [[#3807](https://github.com/airyhq/airy/pull/3807)]
+
+#### 🧰 Maintenance
+
+- Bump @types/node from 18.8.2 to 18.11.4 [[#3872](https://github.com/airyhq/airy/pull/3872)]
+- Bump react-redux from 8.0.2 to 8.0.4 [[#3772](https://github.com/airyhq/airy/pull/3772)]
+- Bump @testing-library/dom from 8.18.1 to 8.19.0 [[#3846](https://github.com/airyhq/airy/pull/3846)]
+- Bump i18next from 21.9.2 to 21.10.0 [[#3847](https://github.com/airyhq/airy/pull/3847)]
+- Bump @babel/preset-env from 7.19.3 to 7.19.4 [[#3848](https://github.com/airyhq/airy/pull/3848)]
+- Bump @pmmmwh/react-refresh-webpack-plugin from 0.5.7 to 0.5.8 [[#3843](https://github.com/airyhq/airy/pull/3843)]
+- Bump @svgr/webpack from 6.4.0 to 6.5.0 [[#3845](https://github.com/airyhq/airy/pull/3845)]
+- Bump babel-jest from 29.1.2 to 29.2.0 [[#3842](https://github.com/airyhq/airy/pull/3842)]
+- Bump cypress from 10.9.0 to 10.10.0 [[#3844](https://github.com/airyhq/airy/pull/3844)]
+- Bump @babel/plugin-proposal-object-rest-spread from 7.18.9 to 7.19.4 [[#3837](https://github.com/airyhq/airy/pull/3837)]
+- Bump @svgr/plugin-svgo from 6.3.1 to 6.5.0 [[#3831](https://github.com/airyhq/airy/pull/3831)]
+- Bump jest-environment-jsdom from 29.1.2 to 29.2.0 [[#3832](https://github.com/airyhq/airy/pull/3832)]
+- Bump minimist from 1.2.6 to 1.2.7 [[#3833](https://github.com/airyhq/airy/pull/3833)]
+- Bump eslint-plugin-react from 7.31.8 to 7.31.10 [[#3828](https://github.com/airyhq/airy/pull/3828)]
+- Bump @reduxjs/toolkit from 1.8.5 to 1.8.6 [[#3808](https://github.com/airyhq/airy/pull/3808)]
+- Bump preact from 10.11.0 to 10.11.2 [[#3829](https://github.com/airyhq/airy/pull/3829)]
+
+#### Airy CLI
+
+You can download the Airy CLI for your operating system from the following links:
+
+[MacOS](https://airy-core-binaries.s3.amazonaws.com/0.51.0/darwin/amd64/airy)
+[Linux](https://airy-core-binaries.s3.amazonaws.com/0.51.0/linux/amd64/airy)
+[Windows](https://airy-core-binaries.s3.amazonaws.com/0.51.0/windows/amd64/airy.exe)
+
```

Comment:
Not sure why the change in the changelog is here? Perhaps a rebase is needed on top of `develop`.